### DISCUSSION TITLE
Welcome to Leetcode75 Discussions!

### DISCUSSION THREAD

**Discussion by strang3-r** [Black-D3vil007/Leetcode75#51] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smartytushar** [comment]
Hello @Black-D3vil007 can I contribute here for Hacktoberfest-2022 ?

**Reply by strang3-r** [reply]
Yes You can Contribute here

**Comment by smartytushar** [comment]
Can I contribute in any language?

**Reply by strang3-r** [reply]
Yes
